sbt-cross/src/main/scala/sbtcross/JvmCross.scala

```diff
@@ -0,0 +1,37 @@
+package sbtcross
+
+import sbt._
+
+case object JVMPlatform extends Platform {
+  def identifier: String = "jvm"
+  def enable(project: Project): Project =
+    project.settings(crossPlatform := JVMPlatform)
+  val crossBinary: CrossVersion = CrossVersion.binary
+  val crossFull: CrossVersion   = CrossVersion.full
+}
+
+trait JvmCross {
+  val JVMPlatform = sbtcross.JVMPlatform
+
+  implicit class JvmCrossProjectBuilder(builder: CrossProject.Builder) {
+    private def build = builder.crossType(CrossType.Full)
+
+    def jvm: Project = build.jvm
+
+    def jvmSettings(ss: Def.SettingsDefinition*): CrossProject =
+      build.jvmSettings(ss: _*)
+
+    def jvmConfigure(transformer: Project => Project): CrossProject =
+      build.jvmConfigure(transformer)
+  }
+
+  implicit class JvmCrossProject(project: CrossProject) {
```

Comment:
I suggest naming this `CrossProjectJVMOps`. The `Ops` suffix is relatively common for implicit classes in Scala.